py/torch_tensorrt/ts/_compile_spec.py

```diff
@@ -225,8 +225,8 @@ def _parse_input_signature(input_signature: Any):
 
 
 def _parse_compile_spec(compile_spec_: Dict[str, Any]) -> _ts_C.CompileSpec:
-    # TODO: Remove deep copy once collections does not need partial compilation
-    compile_spec = deepcopy(compile_spec_)
+    # TODO: Use deepcopy to support partial compilation of collections
+    compile_spec = compile_spec_
```

Comment:
Is this from upstreaming change?
Let's keep this change as a separate PR

Reply:
This is needed for 1.2 to work. By not using deepcopy, I didn't see any issues with input_signature. However, with deepcopy, dataloadercalibrator object can't be pickled was the issue.

Reply:
@peri044 this is a result of not deepcopying https://app.circleci.com/pipelines/github/pytorch/TensorRT/882/workflows/942a4e63-8aeb-44da-a93b-3c2a18cc08f7/jobs/3848/tests

Reply:
We can add deepcopy support for PTQ here: https://pybind11.readthedocs.io/en/stable/advanced/classes.html#deepcopy-support